src/llama4u.py

```diff
@@ -1,111 +1,155 @@
 """ Llama4U """
-import importlib.metadata
 import sys
-import argparse
-from math import exp
-from statistics import median
 from os import devnull
 from contextlib import contextmanager,redirect_stderr
 from termcolor import colored
 from huggingface_hub import hf_hub_download
-from llama_cpp import Llama
+import llama_cpp
+from langchain_community.llms.llamacpp import LlamaCpp
+from langchain.chains.conversation.base import ConversationChain
+from langchain.memory.buffer import ConversationBufferMemory
+from langchain_core.prompts import (
+    ChatPromptTemplate, HumanMessagePromptTemplate
+)
+from input.input import parse_arguments
+
+LLAMA4U_STR = 'Llama4U'
 
 class Llama4U():
     """ Llama4U """
+
+    # Model config parameters
+    model_kwargs = {
```

Comment:
Config param values for model and chat, taken from https://llama-cpp-python.readthedocs.io/en/stable/api-reference/#llama_cpp.Llama.create_completion, which is "invoked" by LangChain's https://api.python.langchain.com/en/latest/chains/langchain.chains.conversation.base.ConversationChain.html#langchain.chains.conversation.base.ConversationChain.predict